generators/server/templates/src/main/java/package/config/_UaaConfiguration.java

```diff
@@ -87,9 +91,8 @@ public void configure(ClientDetailsServiceConfigurer clients) throws Exception {
          */
         clients.inMemory()
             .withClient("web_app")
-            .scopes("web-app")
+            .scopes("openid")
```

Comment:
I don't think this breaks anything. I believe the former "web-app" scope wasn't being used at all in the client. Base64 is for the client_id and client_secret. I do agree that the client_id and client_secret should be encoded via JS rather than hardcoded...but that's a different matter.